k8s/job.go

```diff
@@ -14,6 +14,40 @@ import (
 	yamlutil "k8s.io/apimachinery/pkg/util/yaml"
 )
 
+func ApplyJobExecuterServiceAccount(kubeConfigPath string) error {
```

Comment:
same as above, we can include this in a separate authorization package, better to have also a separate package for rbac